plugins/inputs/dnac/README.md

```diff
@@ -0,0 +1,78 @@
+# Cisco DNAC Input Plugin
+
+The Cisco DNAC plugin gathers metrics about the health of the network monitored
+by DNA Assurance.
+
+## Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage <plugin-name>`.
+
+```toml
+[[inputs.dnac]]
+  ## Specify DNAC Base URL
+  dnacbaseurl = "sandboxdnac.cisco.com"
+  ## Specify Credentials
+  username = "devnetuser"
+  password = "Cisco123!"
+  ## Debug true/false
+  debug = "false"

```

Comment:
we already have a global --debug option. I don't think we should have a local one as well. Can you enable this if the global setting is set?

Reply:
got it, I will use telegraf.Debug to flag whether to set the debug flag in the DNAC client SDK